proto/process/agent.proto

```diff
@@ -651,6 +669,17 @@ message OwnerReference {
 	string kind = 3;
 }
 
+// reference https://github.com/kubernetes/kubernetes/blob/release-1.21/pkg/apis/core/types.go#L4606
+message ObjectReference {
```

Comment:
Are all these fields filled in for active jobs? seems a lot of info

Reply:
Except `fieldPath` which is only relevant when referring to sub-objects all of those will be used